packages/bpk-component-map/readme.md

```diff
@@ -70,12 +71,39 @@ export default () => (
 
 ### BpkMap
 
-| Property	      | PropType	| Required                	| Default Value |
-| --------------- | --------- | ------------------------- | ------------- |
-| mapRef          | func      | false                     | null          |
+| Property	       | PropType                                     | Required                 | Default Value |
+| ---------------- | -------------------------------------------- | ------------------------ | ------------- |
+| containerElement | node                                         | true                     | -             |
+| mapElement       | node                                         | true                     | -             |
```

Comment:
Can we provide default implementations for `containerElement` and `mapElement` ?